docs/userguide/torch_forecasting_models.md

```diff
@@ -313,53 +313,102 @@ to see examples of how to do it.
+| `NBEATSModel` | Energy  | 32    | no   | 32         | 4           | 117s           |
+| `NBEATSModel` | Energy  | 32    | no   | 1024       | 0           | 28.4s          |
+| `NBEATSModel` | Energy  | 32    | no   | 1024       | 2           | 27.4s          |
+| `NBEATSModel` | Energy  | 32    | no   | 1024       | 4           | 27.5s          |
+| `NBEATSModel` | Energy  | 32    | yes  | 32         | 0           | 41.5s          |
+| `NBEATSModel` | Energy  | 32    | yes  | 32         | 2           | 40.6s          |
+| `NBEATSModel` | Energy  | 32    | yes  | 1024       | 0           | 2.8s           |
+| `NBEATSModel` | Energy  | 32    | yes  | 1024       | 2           | 1.65           |
+| `NBEATSModel` | Energy  | 32    | yes  | 1024       | 4           | 1.8s           |
+|               |         |       |      |            |             |                |
+| `TFTModel`    | Energy  | 64    | no   | 32         | 0           | 78s            |
+| `TFTModel`    | Energy  | 64    | no   | 32         | 2           | 72s            |
+| `TFTModel`    | Energy  | 64    | no   | 32         | 4           | 72s            |
+| `TFTModel`    | Energy  | 64    | no   | 1024       | 0           | 46s            |
+| `TFTModel`    | Energy  | 64    | no   | 1024       | 2           | 38s            |
+| `TFTModel`    | Energy  | 64    | no   | 1024       | 4           | 39s            |
+| `TFTModel`    | Energy  | 64    | yes  | 32         | 0           | 125s           |
+| `TFTModel`    | Energy  | 64    | yes  | 32         | 2           | 115s           |
+| `TFTModel`    | Energy  | 64    | yes  | 1024       | 0           | 59s            |
+| `TFTModel`    | Energy  | 64    | yes  | 1024       | 2           | 50s            |
+| `TFTModel`    | Energy  | 64    | yes  | 1024       | 4           | 50s            |
+|               |         |       |      |            |             |                |
+| `TFTModel`    | Energy  | 32    | no   | 32         | 0           | 70s            |
+| `TFTModel`    | Energy  | 32    | no   | 32         | 2           | 62.6s          |
+| `TFTModel`    | Energy  | 32    | no   | 32         | 4           | 63.6           |
+| `TFTModel`    | Energy  | 32    | no   | 1024       | 0           | 31.9s          |
+| `TFTModel`    | Energy  | 32    | no   | 1024       | 2           | 45s            |
+| `TFTModel`    | Energy  | 32    | no   | 1024       | 4           | 44s            |
+| `TFTModel`    | Energy  | 32    | yes  | 32         | 0           | 73s            |
+| `TFTModel`    | Energy  | 32    | yes  | 32         | 2           | 58s            |
+| `TFTModel`    | Energy  | 32    | yes  | 1024       | 0           | 41s            |
+| `TFTModel`    | Energy  | 32    | yes  | 1024       | 2           | 31s            |
+| `TFTModel`    | Energy  | 32    | yes  | 1024       | 4           | 31s            |
+
+-------------
+
+## Early Stop
```

Comment:
Could we maybe put this one higher up? I would keep the benchmarks for the bottom of the page.